src/prompts/grade.ts

```diff
@@ -199,3 +204,31 @@ export function messages(
     createInput(pdfBuffer),
   ];
 }
+
+function hasGmail(flag: string) {
+  const r = new RegExp(/gmail/i);
```

Comment:
Is this going to filter the "don't use hotmail, use gmail instead" flag? That one is important to keep.

Reply:
Didn't think of that scenario that both domains are mentioned, should probably check that it's mentioning Gmail && not hotmail